joinmarket/configure.py

```diff
@@ -252,6 +252,8 @@ def get_blockchain_interface_instance(_config):
         bc_interface = RegtestBitcoinCoreInterface(rpc)
     elif source == 'blockr':
         bc_interface = BlockrInterface(testnet)
+    elif source == 'blockchaininfo':
```

Comment:
I'm slightly concerned about blockchain.info being confused with _the_ blockchain, which the company itself seems to encourage. So maybe it's better to have this configuration be `bci` or `blockchaindotinfo`.